opensfm/reconstruction.py

```diff
@@ -64,6 +64,11 @@ def _add_camera_to_bundle(ba, camera, camera_prior, constant):
             camera.id, camera.focal, camera.k1, camera.k2,
             camera_prior.focal, camera_prior.k1, camera_prior.k2,
             constant)
+    elif camera.projection_type == 'dual':
+        ba.add_dual_camera(
+            camera.id, camera.focal, camera.k1, camera.k2,
+            camera.focal_prior, camera.k1_prior, camera.k2_prior,
```

Comment:
priors are now in `camera_prior`
```suggestion
            camera_prior.focal, camera_prior.k1, camera_prior.k2,
```